samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/ManyGraphics.razor

```diff
@@ -1,7 +1,7 @@
 ﻿@page "/many-graphics"
 
 <PageTitle>Many Graphics</PageTitle>
-<h1>Many Points</h1>
+<h1>Many Graphics</h1>
 <div class="links-div">
     <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-Graphic.html">ArcGIS API for JavaScript</a>
 </div>
```

Comment:
maybe add a quick blurb referencing the one exception for feature layers if whether it is loaded or not. "If the layer is already loaded, you must use <see cref="ApplyEdits"/> to add graphics."

Reply:
In this PR I actually fixed that, I believe. Now `Add` uses `ApplyEdits` under the hood and doesn't fail.